distributions/aws/test/e2e/README.md

```diff
@@ -0,0 +1,51 @@
+
+## Usage
+
+Install dependencies
+```
+python3
```

Comment:
We should eventually have a one line installation for these using either a script or a conda or pip package as possible.